docs/architecture/adr-008-dCERT-group.md

```diff
@@ -0,0 +1,129 @@
+ - compensation stipend from the community pool or other means decided by
+   governance
+
+This system necessitates the following new parameters: 
+ - blockly stipend allowance per dCERT member (suggested ~$10K/yr of atoms)
+ - maximum number of dCERT members (suggested 7) 
+ - required staked slashable tokens for each dCERT member (suggested $10K of atoms)
+ - unbonding time for dCERT staked token (suggested 3 week)
+ - quorum for suspending a particular member (suggested 5/7) 
+ - proposal wager for disbanding the dCERT group (suggested ~$100000 USD worth of atoms)
+
+These parameters are expected to be implemented through the param keeper such 
+that governance may change them at any given point. 
+
+### Continuous Voting Electionator
+
+An `Electionator` object is to be implemented with continuous voting is to be
+implemented with the following specifications:
+ - All delegation addresses may submit votes at any point which updates their 
+   preferred representation on the dCERT group. 
+ - Preferred representation may be arbitrarily split between address (ex. 50%
+   to John, 25% to Sally, 25% to Carol) 
+ - Addresses which control the greatest amount of preferred-representation are
+   eligible to join the dCERT group (up the _maximum number of dCERT members_)
+ - In order for a new member to be added to the dCERT group they must 
+   send a transaction accepting their admission at which point the validity of
+   their admission is to be confirmed. 
+ - If the dCERT group is already full and new member is admitted, the existing
+   dCERT member with the lowest amount of atoms is kicked from the dCERT group. 
+
+### Staking/Slashing
+
+All members of the dCERT group must stake tokens _specifically_ to maintain
+eligibility as a dCERT member. This staking mechanism must be designed with an
+unbonding period. Slash a particular dCERT member due to soft-contract breach
+should be performed by governance on a per member basis based on the magnitude
+of the breach.  The process flow is anticipated to be that a dCERT member is
+suspended by the dCERT group prior to being slashed by governance. 
+
+Additionally in the case of an emergency situation of a colluding and malicious
+dCERT group, the community needs the capability to disband the entire dCERT
+group and likely fully slash them. This could be achieved though a special new
+proposal type (general governance) which would halt the functionality of the
+dCERT group until the proposal was concluded. This special proposal type would
+likely need to also have a fairly large wager which could be slashed if the
+proposal creator was malicious. 
+
+### dCERT membership transactions
+
+Active dCERT members 
+ - change of the description of the dCERT group
+ - circuit break a message route
+ - vote to suspend a dCERT member. 
+
+Here circuit-breaking refers to the capability to disable a groups of messages,
+This could for instance mean: "disable all staking-delegation messages", or
```

Comment:
idk...this seems risky to have 7 members have the ability at will to just circuit break any message. If they do so, then the stakeholders would have to submit a disbandment governance proposal which takes time. Am I missing something here?